### PR TITLE
add: `onnxruntime-bin`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/onnxruntime-bin/onnxruntime-bin.pacscript
Comment (wizard-28) [inline packages/onnxruntime-bin/onnxruntime-bin.pacscript:1]: Add `repology`

**Review by wizard-28 — APPROVED** [review]

File: packages/onnxruntime-bin/onnxruntime-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+maintainer="Zahrun <Zahrun@github.com>"
+
+install() {
+  sudo install -Dm755 "lib/lib${pkgname}.so.${version}" -t "${pkgdir}/usr/local/lib"
+  sudo install -Dm644 include/* -t "${pkgdir}/usr/local/include"
+  sudo install -Dm644 LICENSE -t "${pkgdir}/usr/share/licenses/${name}"
+}
+
+postinst() {
+  sudo rm -f "/usr/local/lib/lib${pkgname}.so" 2> /dev/null
```
Comment (Henryws) [inline packages/onnxruntime-bin/onnxruntime-bin.pacscript:16]:
```suggestion
  sudo rm -f "/usr/local/lib/lib${pkgname}.so"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by wizard-28 — DISMISSED** [review]

File: packages/onnxruntime-bin/onnxruntime-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name="onnxruntime-bin"
+pkgname="onnxruntime"
+repology=("project: onnxruntime")
+version="1.14.1"
+url="https://github.com/microsoft/onnxruntime/releases/download/v${version}/onnxruntime-linux-x64-${version}.tgz"
+description="ONNX Runtime is a cross-platform inference and training machine-learning accelerator."
```
Comment (Henryws) [inline packages/onnxruntime-bin/onnxruntime-bin.pacscript:6]:
```suggestion
description="Cross-platform inference and training machine-learning accelerator"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/onnxruntime-bin/onnxruntime-bin.pacscript (hunk 2 above)
Comment (Zahrun) [reply packages/onnxruntime-bin/onnxruntime-bin.pacscript]: Sure

**Review by Henryws — APPROVED** [review]

**Pull request merged**